src/kOS/Suffixed/AddonList.cs

```diff
@@ -0,0 +1,33 @@
+﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+
+namespace kOS.Suffixed
+{
+    public class AddonList : Structure
+    {
+        private readonly Addon kacAddon;
+        private readonly Addon rtAddon;
+        private readonly Addon agxAddon;
+
+        public AddonList()
+        {
+            kacAddon = new Addon("KAC");

```

Comment:
Yes, some better structure is needed here as we'll have at least 2 more addons soon (IR, Trajectories)
I'll come up with something better tomorrow.